styles/formative/catppuccin.user.css

```diff
@@ -81,1054 +81,665 @@
-            }
-          }
         }
       }
-    }
-    .iCrZor {
-      &.selectedTab {
-        background-color: @accent-color;
-        color: @contrast-accent-color;
+      .ChoiceElement__RootDiv-sc-1mbtxfk-1 {
+        border-radius: 4px;
+        background: @surface0;
       }
-      color: @accent-color;
-    }
-    .fJjhRQ:hover {
-      background-color: @surface1;
-    }
-    /*************************************************************
-     * Join Code
-     *************************************************************/
-    .App__AppContainerDiv-sc-1x6i86e-0 {
-      background-color: @base;
-      .SplitColumnLayout__FullScreenDiv-sc-4mcl10-5 {
-        background-color: @base;
-        svg {
-          path:nth-child(1) {
-            fill: @peach !important;
-            stroke: @peach !important;
-          }
-          path:nth-child(2) {
-            fill: @peach !important;
-            stroke: @peach !important;
-          }
-          path:nth-child(3) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(4) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(5) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(6) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(7) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(8) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(9) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(10) {
-            fill: @green !important;
-            stroke: @green !important;
-          }
-          path:nth-child(11) {
-            fill: @text !important;
-            stroke: @text !important;
-          }
+      .ChoiceElement__StyledRawButton-sc-1mbtxfk-0 {
+        .ChoiceElement__LabelRichText-sc-1mbtxfk-3,
+        .material-icons-outlined {
+          background: transparent;
         }
       }
-      .Elements__ColumnDiv-sc-1qtb2ho-2,
-      .Elements__TwoColumnParentDiv-sc-1qtb2ho-1 {
-        background-color: @base;
-      }
-      .SplitColumnLayout__Card-sc-4mcl10-0 {
-        background-color: @mantle;
-        .lmmjCy,
-        .EaPMD,
-        .cuNCkB {
-          color: @text;
+      .DrawingCardButton__RootButton-sc-u2qprg-1 {
+        border-color: @overlay0;
+        .DrawingCardButton__DrawingPreview-sc-u2qprg-3 {
+          background: rgb(245, 246, 248);
         }
-        .JoinQuickCode__ErrorDiv-sc-1uhfett-4 {
-          color: @red;
+        .DrawingCardButton__HoverDarkScrim-sc-u2qprg-2 {
+          background: fade(@crust, 75%);
+          border-radius: 0;
```

Comment:
```suggestion
```

Reply:
Border-radius here is used to fix this small white area in the drawing preview. Is it not allowed to do so?
![image](https://github.com/catppuccin/userstyles/assets/45221816/b5e02412-87a1-4054-b7f2-c508784fb3b1)